src/lib/detection_algorithms/config.json

```diff
@@ -0,0 +1,4 @@
+{
+    "YOLO": "yolo",
+    "CNTK": "cntk"  
```

Comment:
Understood at the time i figured CNTK was using a standard format I see now that the format is specific to this code.